tests/test_misc.py

```diff
@@ -2319,6 +2319,15 @@ def test_emergencyrecover(node_factory, bitcoind):
     listfunds = l1.rpc.listfunds()["channels"][0]
     assert listfunds["short_channel_id"] == "1x1x1"
 
+    # Since reconnect is not allowed in DEV mode...
+    if not DEVELOPER:
+        time.sleep(5)
+        l2.bitcoin.generate_block(5)
+        sync_blockheight(bitcoind, [l1, l2])
+	# Check if funds are recovered.
+        assert l1.rpc.listfunds()["channels"][0]["state"] == "FUNDING_SPEND_SEEN"
```

Comment:
```suggestion
        sync_blockheight(bitcoind, [l1, l2])
	    # Check if funds are recovered.
        assert l1.rpc.listfunds()["channels"][0]["state"] == "FUNDING_SPEND_SEEN"
```

welcome in python fancy compiler parsing :) I think this should fix the CI

Reply:
PS: the UI seems wrong here, but I see in the files changes a different think! also the line of the check `2327` is the same, so I believe that this is the CI problem